app/assets/javascripts/runtime.js

```diff
@@ -41,7 +41,6 @@
 //= require video
 //= require interactive-sizing
 //= require embeddable-carousel
-//= require image_question
```

Comment:
Just noticed `image_question` was removed. 

I guess it was superseded by `image_question_drawing_tool`?  Maybe it is something that should have been removed a while ago?

Reply:
Exactly, it should have been removed long time ago. That was an old version that was using svg-edit editor.